cmake/DLAF_AddTest.cmake

```diff
@@ -171,6 +171,8 @@ function(DLAF_addTest test_target_name)
 
     set(_TEST_COMMAND ${MPIEXEC_EXECUTABLE} ${MPIEXEC_NUMPROC_FLAG} ${DLAF_AT_MPIRANKS} ${_MPI_CORE_ARGS}
         ${MPIEXEC_PREFLAGS} $<TARGET_FILE:${test_target_name}> ${MPIEXEC_POSTFLAGS})
+
+    set(_TEST_LABEL MPI)
   # ----- Classic test
   else()
     set(_TEST_COMMAND ${test_target_name})
```

Comment:
What about setting a label here as well? (E.g. NOMPI)

Reply:
I've added this, but later refactored the shell script not to make use of it. It still there since it cannot hurt to have labels...